### DISCUSSION TITLE
Slower transcoding on 10.7.0 (36 fps -> 10 fps)

### DISCUSSION THREAD

**Discussion by plcharriere** [jellyfin/jellyfin#5419] [General]
Hello,

I noticed a performance change on transcoding with the new update.

On the 10.6.4 version, I get transcoding at 36 fps while on 10.7.0 I get only 10 fps.

10.6.4 : 

![unknown](https://user-images.githubusercontent.com/48135922/110401891-bc647e00-807a-11eb-93f4-7afa1ae8a902.png)

10.7.0 : 

![unknown (1)](https://user-images.githubusercontent.com/48135922/110401840-a3f46380-807a-11eb-9e92-4d9b8a801678.png)

Is it because it now transcodes audio ?

Is this an issue or have I done something wrong ?

Thanks.

**Comment by nyanmisaka** [comment]
Can you share the transcoding log?
